docs/README.md

```diff
@@ -1,10 +1,18 @@
 # Containerd website
 
-The containerd website is built using Jekyll and published to Github pages.
+The containerd website is built using [Hugo](https://gohugo.io) and published to Github pages.
 
-In order to build and test locally:
+To build the site locally (using Docker):
+
+```bash
+$ docker run -it -v $(pwd):/src -p "1313:1313" jojomi/hugo
```

Comment:
Is `-it` required?
Also, probably `--rm` is expected?

Reply:
I don't think that there's really a use case for just building the site using Docker, so I removed these instructions and left only the instructions below for "watch" mode.